doc/api/vm.md

```diff
@@ -304,29 +304,6 @@ console.log(util.inspect(sandbox));
 // { globalVar: 1024 }
 ```
 
-## vm.runInDebugContext(code)
-<!-- YAML
-added: v0.11.14
--->
```

Comment:
Maybe also add a line like `> Stability: 0 - Deprecated: short reason` after the YAML block.